sympy/ntheory/digits.py

```diff
@@ -17,12 +17,37 @@ def digits(n, b=10):
     >>> from sympy.ntheory.digits import digits
     >>> digits(35)
     [10, 3, 5]
-    >>> digits(27, 2)
+    >>> digits(27, 2, None)
     [2, 1, 1, 0, 1, 1]
-    >>> digits(65536, 256)
+    >>> digits(65536, 256, None)
     [256, 1, 0, 0]
-    >>> digits(-3958, 27)
+    >>> digits(-3958, 27, 1)
     [-27, 5, 11, 16]
+    >>> digits(-3958, 27, 2)
+    [-27, 5, 11, 16]
+    >>> digits(35, 10, None)
+    [10, 3, 5]
+    >>> digits(35, 10, 1)
+    [10, 3, 5]
+    >>> digits(35, 10, 2)
+    [100, 3, 5]
+    >>> digits(35, 10, 3)
+    [1000, 3, 5]
+    >>> digits(35, 10, 4)
+    [10000, 3, 5]
+
+    Parameters
+    ==========
+
+    n = integer
+    The number whose list of digits is Computed
```

Comment:
no need for capital C on Computed

Reply:
Sir, I have already commited them. Is it okay if they stay? They are just comments after all.